### DISCUSSION TITLE
group raw datasets in a viz

### DISCUSSION THREAD

**Discussion by bensmith07** [kedro-org/kedro#2040] [Idea]
I would like my visualization to group all raw datasets into one expandable "node" called "raw data", similar to the way that namespaces are collapsible/expandable. From what I can tell, in order to do this via namespace, you must have a function with inputs and outputs, but that would not be appropriate here. 

Here is my current visualization:
![image](https://user-images.githubusercontent.com/96085296/202926578-912cfa25-4205-40e4-b694-7a895ea0e977.png)

I would like the raw datasets to be combined into one node-like object in this visualization.

Thank you!

**Comment by stichbury** [comment]
Hi @bensmith07 -- sorry you've been waiting for an answer. Please head over to the Slack organisation (https://slack.kedro.org) for future queries since you'll get a much more rapid response.
